Client/src/Assets/emptyJSFile.json

```diff
@@ -0,0 +1,96 @@
+{
+    "reference": {
+        "type": "",
+        "publisher": "",
+        "authors": [
+            {
+                "firstname": "",
+                "middlename": "",
+                "lastname": ""
+            },
+            {
+                "firstname": "",
+                "middlename": "",
+                "lastname": ""
+            }
+        ],
+        "title": "",
+        "volume": 0,
+        "pages": 0,
+        "year": 0
+    },
+    "dataset name": "",
+    "material": [
+        {
+            "composition": "",
+            "details": ""
+        },
+        {
+            "composition": "",
+            "details": ""
+        }
+    ],
+    "category": "",
+    "subcategory": "",
+    "data type": "",
+    "data": {
+        "variables": [
+            {
+                "name": "initial density",
```

Comment:
remove the values for name, repr and units, also leave just one variable here

Reply:
done in the latest commit